app/safe/internal/bootstrap/bootstrap.go

```diff
@@ -173,7 +173,7 @@ func CreateCryptoKey(id *string, updatedSecret chan<- bool) {
 
 	if secret != state.BlankAgeKeyValue {
 		log.InfoLn(id, "Secret has been set in the cluster, will reuse it")
-		state.SetMasterKey(secret)
+		state.SetRootKey(secret)
```

Comment:
We use “root key” everywhere.

Renamed it for consistency (and also for inclusivity).